### PR TITLE
[release 2.2] update org references to stolostron

### PR DESCRIPTION
**Pull request by SherinV**
**Related Issue:**  open-cluster-management/backlog#18729

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,11 +6,23 @@ BINDIR ?= output
...
+
+# GITHUB_USER containing '@' char must be escaped with '%40'
+GITHUB_USER:=$(shell echo $(GITHUB_USER) | sed 's/@/%40/g')
+GITHUB_TOKEN?=
+
+GOPRIVATE=github.com/stolostron
+
+ifdef GITHUB_TOKEN
+$(shell git config --global url."https://$(GITHUB_TOKEN):x-oauth-basic@github.com/".insteadOf "https://github.com/")
+endif
```
Comment (jlpadilla) [inline Makefile:24]: I'm curious why none of the other PRs have this change.

**Review by jlpadilla — APPROVED** [review]
/lgtm
/hold

File: Makefile (hunk 1 above)
Comment (SherinV) [reply Makefile:24]: After 2.2, the endpoint-operator repo was moved to klusterlet-addon and is public. But, that change is not ported over to 2.2. endpoint-operator is an indirect dependency for search-collector but is a private repo. So, to pull it had to add this similar to app team's PR https://github.com/stolostron/multicloud-operators-placementrule/tree/release-2.2.

Comment (jlpadilla) [reply Makefile:24]: Thanks! we had a similar line in the .travis file.

**Pull request merged**